### PR TITLE
Rename `gesture` -> `action`

### PR DESCRIPTION
**Pull request by microbit-grace**

### CODE REVIEW

File: src/store.ts [hunk 1]
```diff
@@ -119,7 +119,8 @@ const updateProject = (
 };
 
 export interface State {
-  gestures: GestureData[];
+  // TODO: Rename store gestures as actions instead
+  gestures: ActionData[];
```
Comment (microbit-grace) [inline src/store.ts:123]: This affects the key used in localStorage for storing action data. Renaming to be done in a separate PR as suggested by Matt.

**Review by microbit-matt-hillsdon — APPROVED** [review]
